lib/rules/no-get-with-default.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const types = require('../utils/types');
+
+function makeErrorMessageForGetWithDefault(property, isImportedGetWithDefault) {
+  return isImportedGetWithDefault
```

Comment:
I suggested these two error messages for you to choose from but didn't mean for you to use both of them haha. Just need to pick one. Honestly, it's probably simpler at this point to use a single static string error message in all cases instead of having a function for generating this. Let's use replace this function with:
```
const ERROR_MESSAGE = 'Use `\|\|` or the ternary operator instead of `getWithDefault()`';
```